plugin/src/main/java/com/exadel/aem/toolkit/plugin/handlers/widgets/OptionProviderHandler.java

```diff
@@ -48,7 +49,8 @@ abstract class OptionProviderHandler {
      */
     boolean hasProvidedOptions(OptionProvider optionProvider) {
         boolean hasExternalOptions = ArrayUtils.isNotEmpty(optionProvider.value())
-            && Arrays.stream(optionProvider.value()).anyMatch(source -> StringUtils.isNotBlank(source.value()));
+            && Arrays.stream(optionProvider.value())
```

Comment:
Do we need to handle a situation when user has not provided neither `value` nor `classValue`? Before, `value` was required. Now it looks like a pathway for invalid input. Usually we validate user inputs to prevent  invalid ones, e.g. if the number of columns in a text field can be set up in an annotation, and user enters `-1`, we throw a ValidationException